### PR TITLE
Integrate new icon for TIDAL

### PR DESCRIPTION
**Pull request by myopenflixr**
Updated custom-brand-icons.js.
Added tidal-logo.svg to icon-svg folder
Added icon to README.md

### CODE REVIEW

**Comment by myopenflixr** [comment]
This is my first ever github contribution. Hoping I did everything correctly!

File: README.md [hunk 1]
```diff
@@ -615,6 +616,10 @@ let's take the first string for example:
 
 * `String` = this data can be recovered from the svg file  `<path d="M21,12.5 C21,13.33 18.76,...."` In particular you will have to enter only the part of the vector code `"M21,12.5 C21,13.33 18.76"` For an example take a look at the [icons already inserted](dist/custom-brand-icons.js)
 
+* **When you open a pull request do it on the dev branch**
+
+* In addition to inserting the icon in the `icon-svg` folder and updating the `custom-brand-icons.js` file, remember to also update the `README.md` file by inserting the new icon
+
```
Comment (elax46) [inline README.md:622]: Eliminate this change as in the last commit yesterday I already proceeded to insert a warning on pull requests

**Review by elax46 — CHANGES_REQUESTED** [review]
Why are there several icons deleted and then added? the pull must contain only the three modified files nothing else

File: README.md (hunk 1 above)
Comment (myopenflixr) [reply README.md:622]: I have no idea how that happened. I pulled from the dev branch and only modified the three files. Very strange to see all the deletions and additions. I swear I didn't modify those!

Can you just cancel this request so I can start over and open a new pull request? This is my first time attempting to contribute to a github project, so this is all new to me.

Once this is cancelled so I can start fresh, should I pull from dev or master?

Comment (elax46) [reply README.md:622]: if the changes have been made in your fork, just reopen a pull request, remember to delete what I indicated first

**Pull request closed without merging**